1-exercises/C-while-loop-with-array/exercise.js

```diff
@@ -5,19 +5,29 @@
 */
 
 const BIRTHDAYS = [
-    "January 7th",
-    "February 12th",
-    "April 3rd",
-    "April 5th",
-    "May 3rd",
-    "July 11th",
-    "July 17th",
-    "September 28th",
-    "November 15th"
+  "January 7th",
+  "February 12th",
+  "April 3rd",
+  "April 5th",
+  "May 3rd",
+  "July 11th",
+  "July 17th",
+  "September 28th",
+  "November 15th",
 ];
 
 function findFirstJulyBDay(birthdays) {
-    // TODO
+  let i = 0;
+  let result = [];
+  while (i < birthdays.length) {
+    if (birthdays[i].includes("July")) {
+      let julyNumbers = parseInt(birthdays[i].replace(/\D/g, ""));
+      result.push(julyNumbers);
+    }
+    i++;
+  }
+  return `July ${Math.min(...result)}th`;
```

Comment:
Hi! I actually did this exercise using another loop. Through your solutions, I understood how is done by using a while-loop. Thanks!